### PR TITLE
feat: add Rspack support

### PR DESCRIPTION
**Pull request by chihab**

### CODE REVIEW

File: examples/package.json [hunk 1]
```diff
@@ -21,7 +22,8 @@
     "serve": "^14.2.3",
     "verdaccio": "^5.32.2",
     "webpack": "5.94.0",
-    "webpack-cli": "^5.1.4"
+    "webpack-cli": "^5.1.4",
+    "@rspack/cli": "^1.0.10"
```
Comment (Gehbt) [inline examples/package.json]: Add `@rspack/core` dep for `DefinePlugin`

```suggestion
    "@rspack/cli": "^1.0.10",
    "@rspack/core": "^1.0.10"
```

**Comment by Gehbt** [comment]
I am also find some problem in npm script, I will create a new [PR](https://github.com/chihab/dotenv-run/pull/106) (feat-rspack branch) to fix it.

**Comment by chihab** [comment]
@Gehbt are we good to merge?

**Comment by Gehbt** [comment]
All is fine. And thanks for your support!

**Comment by chihab** [comment]
Thanks a lot of your time and your contribution.
It would be cool to have the usage documented on the README.md

**Pull request merged**